fltkhs.cabal

```diff
@@ -12,29 +12,62 @@ bug-reports: https://github.com/deech/fltkhs/issues
 copyright: © 2017 Aditya Siram All Rights Reserved
 category: UI,Graphics,GUI,User Interfaces
 build-type: Custom
-cabal-version: >=1.24
+cabal-version: >=2.0
+  -- for custom-setup 
 source-repository head
   type: git
   location: http://github.com/deech/fltkhs
 
-Flag Bundled
+----------------------------------------
+-- by default, 
+-- 
+-- * no OpenGL support
+-- * the system `fltk` package is used (not the bundled tarball)
+--
+
+Flag bundled
    Description: Use the bundled FLTK library.
    Manual: True
    Default: False
 
-Flag Opengl
+Flag opengl
    Description: Enable OpenGL support.
    Manual : True
    Default: False
 
+Flag executables
+   Description: Build executables.
+   Manual : True
+   Default: True
+
+----------------------------------------
+
 custom-setup
   setup-depends:
     base >= 4.5 && < 4.11,
     Cabal >= 1.24.1 && < 1.25,
     filepath,
     directory >= 1.2.3.0
 
+----------------------------------------
+
 library
+
+  extra-libraries: jpeg
+      -- from the `libjpeg` system package 
+
+  build-tools: autoconf, c2hs
```

Comment:
For some reason `autoconf` isn't recognizes as a build tool in Cabal, even though it is on my path.